### DISCUSSION TITLE
Xiaomi Smart Standing fan 2 Pro

### DISCUSSION THREAD

**Discussion by Rafas20** [syssi/xiaomi_fan#258] [General]
I tried to integrate my new fan to HA but it does not show any controls for the fan.. Did i made anything wrong? Could you help me please? 
![Screenshot_22](https://github.com/syssi/xiaomi_fan/assets/143114577/3c9a6910-9878-467a-95e9-90c31a82e67b)
![Screenshot_11](https://github.com/syssi/xiaomi_fan/assets/143114577/8f5b38e0-a041-46d3-bd76-b94410799998)

**Comment by freenandes** [comment]
I can't even get it to show up like you did... Exactly the same fan. I just have it in the config file, like yours. Did that happen to you? How did you solve it? I tried rebooting multiple times, still no device.

Do you have full control over it now?

**Reply by Rafas20** [reply]
yes... Search it on Devices & Services then go to entities and type "fan" in the search box and then you will see the controls of it
